cmake/linux/debian/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+SET(CPACK_DEBIAN_PACKAGE_VERSION "${VERSION}")
+SET(CPACK_DEBIAN_PACKAGE_NAME "${CMAKE_PROJECT_NAME}")
+SET(CPACK_DEBIAN_PACKAGE_SECTION "sound")
+SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "${PROJECT_AUTHOR} <${PROJECT_EMAIL}>")
+set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "${PROJECT_URL}")
+SET(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "any")
+SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "${PROJECT_DESCRIPTION}")
+SET(CPACK_DEBIAN_PACKAGE_RECOMMENDS "${CPACK_DEBIAN_PACKAGE_NAME}-vst-full (= ${DOLLAZ}{binary:Version}) [amd64 i386], tap-plugins, caps")
+SET(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_NAME}-common (= ${DOLLAZ}{source:Version}), ${DOLLAZ}{shlibs:Depends}, ${DOLLAZ}{misc:Depends}, zynaddsubfx | zynaddsubfx-git")
+SET(CPACK_DEBIAN_PACKAGE_SUGGESTS  "fil-plugins, mcp-plugins, omins, freepats, fluid-soundfont-gm")
+SET(CPACK_DEBIAN_PACKAGE_DEPENDS " cmake,
+ debhelper (>= 9.0.0),
+ fluid,
+ gcc-multilib [amd64],
+ g++-multilib [amd64],
+ imagemagick,
+ ladspa-sdk,
+ libsdl-sound1.2-dev,
+ libsamplerate0-dev,
+ libsndfile1-dev,
+ libvorbis-dev,
+ libjack-dev,
+ libstk0-dev,
+ libflac-dev,
+ libfltk1.3-dev,
+ libxinerama-dev,
+ libasound2-dev [linux-any],
+ libqt4-dev,
+ libpulse-dev,
+ libfluidsynth-dev,
+ libfftw3-dev,
+ libfreetype6-dev,
+ libxft-dev,
+ portaudio19-dev,
+ qt4-qmake, 
+ wine-dev[i386]")
+SET(CPACK_PACKAGE_DESCRIPTION " ${PROJECT_NAME_UCASE} aims to be a free alternative to popular (but commercial and closed-
+ source) programs like FruityLoops, Cubase and Logic giving you the ability of
+ producing music with your computer by creating cool loops, synthesizing and
+ mixing sounds, arranging samples, having more fun with your MIDI-keyboard
+ and much more...
+ .
+ ${PROJECT_NAME_UCASE} combines the features of a tracker-/sequencer-program (pattern-/channel-/
+ sample-/song-/effect-management) and those of powerful synthesizers and
+ samplers in a modern, user-friendly and easy to use graphical user-interface")
+SET(ICON_INSTALL "data/lmms.png")
```

Comment:
Like above, not mandatry, but we should use some naming convention here that make sense.  `ICON_INSTALL` is certainly better than `ICON`, but if the name can help explain context, that helps as I've no clue the difference between the different icon variables at a glance.